v0-991/learn/tools-ides/intellij-plugin.md

```diff
@@ -101,28 +104,28 @@ After obtaining the ZIP file using either of the above approaches, follow the st
 
 ## Using the plugin
 
-For information on using the IntelliJ Ballerina plugin to write Ballerina programs, see [Using the IntelliJ Ballerina plugin](tools-ides/intellij-plugin/using-the-intellij-plugin).
+For information on using the IntelliJ Ballerina plugin to write Ballerina programs, see [Using the IntelliJ Ballerina plugin](/learn/tools-ides/intellij-plugin/using-the-intellij-plugin).
```

Comment:
link should start with the version related to the page ... so here the link should be `/v0-991/learn/tools-ides/intellij-plugin/using-the-intellij-plugin`

Reply:
Checked and fixed in the below pages:

- /learn/tools-ides/intellij-plugin.md
- v0-991/learn/tools-ides/intellij-plugin.md
- v1-0/learn/tools-ides/intellij-plugin.md
- v1-1/learn/tools-ides/intellij-plugin.md